portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,11 +80,56 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty(COMMENT_TEXT, commentText);
     commentEntity.setProperty(TIME_STAMP, timeStamp);
 
+    // TODO: Remove print statement and store image URL in database.
+    System.out.println(getUploadedFileUrl(request, /* formInputElementName= */ "image").get());
+
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(commentEntity);
 
     response.sendRedirect(REDIRECT_URL);
   }
+
+  /** 
+   * Returns a URL that points to the uploaded file, or an empty optional
```

Comment:
```suggestion
   * Returns the URL that points to the uploaded file, or an empty optional
```

Reply:
This JavaDoc doesn't explain what the uploaded file is? Is it {@code formInputElementName}. It also doesn't explain what the arguments are.

Reply:
fixed